plugins/chat/app/services/chat_publisher.rb

```diff
@@ -230,6 +230,8 @@ def self.publish_chat_channel_metadata(chat_channel)
     )
   end
 
+  CHANNEL_ARCHIVE_STATUS_BUS_CHANNEL = "/chat/channel-archive-status"
```

Comment:
```suggestion
  CHANNEL_ARCHIVE_STATUS_MESSAGE_BUS_CHANNEL = "/chat/channel-archive-status"
```

For consistency with the other constants.